String/Aufgabe5Palindrom.java

```diff
@@ -0,0 +1,43 @@
+package QC_Java.String;
+
+public class Aufgabe5Palindrom {
+    /*
+    Aufgabe 5
+    Implementieren Sie eine statische Methode palindrom, die aus einem Text alle
+    Palindrome auf der Konsole ausgibt. Folgender Aufruf
+    palindrom("Der vor dir stehende Rentner heißt Otto Uwe") sollte z.B. folgendes
+    auf der Konsole ausgeben:
+    Rentner
+    Otto
+    Verwenden Sie die folgende Signatur für die Methode:
+    public static void palindrom(String text)
+     */
+    public static  void splitStringInArray(){
```

Comment:
The name of the method doesn't reflect the action it does. I would name it as `printPolyndroms` or something like that. Also it should accept a String, a sentence which it parses and prints polyndromes.

Reply:
Fixed